cla_socketserver/app.js

```diff
@@ -31,51 +27,27 @@ app.use(function(err, req, res, next){
 });
 
 // ADMIN
-
-adminApp.install(app, nsp);
-
+adminApp.install(app, socketServer);
 
 // SOCKETS
+const socketNamespace = socketServer.of('/socket.io');
```

Comment:
This is all ES5, so you'll have to use `var` instead of `const`. We're not using Bable or any other types of compilers to make it backwards compatible. 

You've used `const` in a compile of places, so just change them to `var`.

Reply:
Another reason to swop out gulp or modify it to actual use Bable so we can use the magic of ES6